src/Dropdown/Dropdown.js

```diff
@@ -19,23 +19,60 @@ export default class Dropdown extends Util.mixin(BindMixin) {
   constructor() {
     super();
     this.state = {
+      menuDirection: 'down',
+      menuHeight: null,
       isOpen: false,
-      selectedID: null
+      selectedID: null,
+      knowMenuHeight: false
```

Comment:
Do we need this property?
Wouldn't it be good enough to check if `menuHeight` is not `null`?